### DISCUSSION TITLE
Cuddle With Monika Submod

### DISCUSSION THREAD

**Discussion by eyelesssally** [Monika-After-Story/MonikaModDev#7818] [general - submod / external content]
This mod would allow you to cuddle up with Monika and just chill together. It would work kind of like the "holding" mechanic. If you have any coding or art skills, I would love to do this with you! See the reddit post [here](https://old.reddit.com/r/MASFandom/comments/jl5r6i/wip_submod_sleepycuddle_time_with_monika/).

![sleepytimetest](https://user-images.githubusercontent.com/56178333/97764267-730c8c80-1ae4-11eb-8cdf-a67339f78704.png)

**Comment by multimokia** [comment]
We have this planned for bedroom mode officially.

**Comment by Jojojihn** [comment]
> We have this planned for bedroom mode officially.

May I ask for when the bedroom mode is planned?

**Comment by fromulus** [comment]
> May I ask for when the bedroom mode is planned?

Some point before the heat death of the universe.

**Reply by Aem1024** [reply]
> > May I ask for when the bedroom mode is planned?
> 
> Some point before the heat death of the universe.

Ah. Well at least monika will be able to sleep with my corpse.

**Reply by MetalBrother** [reply]
> > > May I ask for when the bedroom mode is planned?
> > 
> > 
> > Some point before the heat death of the universe.
> 
> Ah. Well at least monika will be able to sleep with my corpse.

Brutal \m/

**Comment by scolotron5** [comment]
Welp

**Comment by xXMidnight** [comment]
> > May I ask for when the bedroom mode is planned?
> 
> Some point before the heat death of the universe.

What makes you think it'll be a heat death and not an expansion death

**Comment by MiguelFitz** [comment]
> > > May I ask for when the bedroom mode is planned?
> > 
> > 
> > Some point before the heat death of the universe.
> welp, i guess im gona take a sit and wait 
> What makes you think it'll be a heat death and not an expansion death

**Comment by ironhearxt** [comment]
what..?

**Reply by ExplorerKiller** [reply]
i could make a hoodie for monika and a cute table cat stand for monika on PhotoPea
